### PR TITLE
Fix build warnings

### PR DESCRIPTION
**Pull request by remicollet**

### CODE REVIEW

**Comment by remicollet** [comment]
Perhaps more to come...

File: usefulstuff.c [hunk 1]
```diff
@@ -619,7 +619,7 @@ int xdebug_format_output_filename(char **filename, char *format, char *script_na
 				case 'R': { /* $_SERVER['REQUEST_URI'] */
 					char *char_ptr, *strval;
 #if PHP_VERSION_ID >= 70000
-					zval *data;
+					zval *data=NULL;
```
Comment (derickr) [inline usefulstuff.c]: Can you please stick to the coding standards? e.g, spaces around the "=":

```
zval *data = NULL;
```

**Comment by remicollet** [comment]
Can you have a look at 

     /work/GIT/xdebug/xdebug_var.c:2807:9: warning: 'tmp_target' may be used uninitialized in this function [-Wmaybe-uninitialized]
       retval = xdebug_sprintf("<a href='%s%s%s' target='_new'>%s</a>\n",
              ^

Other seems false positive

**Comment by derickr** [comment]
I guess it was theoretically possible to use tmp_target uninitialized, so I've fixed it. I've also merged the rest of this PR. Thanks again!

**Pull request closed without merging**